atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -42,7 +42,7 @@
     private static final int GET_RANGES_QUEUE_SIZE_WARNING_THRESHOLD = 1000;
 
     public static final Logger log = LoggerFactory.getLogger(AbstractTransactionManager.class);
-    protected final TimestampCache timestampValidationReadCache = TimestampCache.create();
+    protected final TimestampCache timestampValidationReadCache = new TimestampCache();
```

Comment:
Should we create with initial desired size in constructor?

Reply:
it's difficult, working around some architectural problems because things in AbstractTransactionManager can't very easily get anything from an atlas config (notably, some ways to get into this code don't even have an atlas config)